packages/reporter-vue/src/runnables/Runnable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div :class="`runnable-root ${state}-state`">
+    <BaseAccordion v-model="open">
+      <template #header>
+        <div data-cy="title" :class="`runnable-title-wrapper hover:bg-warm-gray-50`">
+          <div :class="`transform runnable-title ${runnable.type}-title`">
+          <i-fa-caret-right v-if="type === 'suite'" :class="`${open && 'rotate-90'} transform text-warm-gray-400`" />
+          <template v-else>
+            <i-fa-solid-times v-if="state ==='failed'"/>
+            <i-fa-circle-o-notch v-if="state ==='pending'"/>
+            <i-fa-check v-if="state === 'passed'"/>
+            <i-mdi-square-outline v-if="state === 'not-started'"/>
+            <i-fa-refresh v-if="state === 'processing'" class="animate-spin-slow"/>
+          </template>
+          {{ runnable.title }}
+          </div>
+        </div>
+      </template>
+      <!-- Inner content of the suite -->
+      <div data-cy="content" class="ml-0.4rem"><slot name="default" /></div>
+    </BaseAccordion>
+  </div>
+</template>
+
+<script lang="ts" >
+import { defineComponent, computed, PropType, ref } from 'vue'
+import BaseAccordion from '../components/BaseAccordion.vue'
+import { Test, Suite } from '../store/reporter-store';
+
+export default defineComponent({
+  components: { BaseAccordion },
+  props: {
+    runnable: {
+      type: Object as PropType<Test | Suite>,
+      required: true
+    },
+  },
+  setup(props) {
+    return {
+      open: ref(true),
+      state: computed(() => props.runnable.state),
```

Comment:
Are you doing `computed` here because it's required to get Pinia to be reactive?

Pinia seems a bit buggy; I really don't expect to explicitly make my `props` reactive by using `computed`.